src/Analysis/Core/Impl/Interpreter/InterpreterConfiguration.cs

```diff
@@ -184,7 +184,7 @@ public static InterpreterConfiguration FromDictionary(IReadOnlyDictionary<string
         /// <summary>
         /// Module cache folder.
         /// </summary>
-        public string ModuleCachePath { get; set; }
+        public string DatabasePath { get; set; }
```

Comment:
I assume this isn't a breaking change? (I think so, given https://github.com/Microsoft/vscode-python/blob/9187479ea42963d28640d8abaaa7345df77c0c7b/src/client/activation/languageServer/analysisOptions.ts#L79)

Reply:
Client passes it in property dictionary in
```ts
properties['DatabasePath']
```
which we read via 
```cs
DatabasePath = Read(properties, nameof(DatabasePath));
```
 so it should be OK.